### PR TITLE
Integrate wix latest

### PR DESCRIPTION
**Pull request by tonimelisma**
First step is to bring your repo up to date with all the updates from wix. Not sure which PRs you already integrated into your repos but I think several of them (HEIF support etc) were overlapping with the wix changes.

These changes also include one or two commits in between that aren't compatible with standard libvips, they're linked to changes they've done to their own version which haven't yet been propagated upstream. I can fix them later.

### CODE REVIEW

**Comment by tonimelisma** [comment]
And, naturally, the go.mod URL needs to be fixed.

File: README.md [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+- [libvips](https://github.com/libvips/libvips) 8.10+
 - C compatible compiler such as gcc 4.6+ or clang 3.0+
-- Go 1.9+
+- Go 1.14+
 
-# Installation
+## Installation
 ```bash
-go get -u github.com/davidbyttow/govips/pkg/vips
+go get -u github.com/wix-playground/govips/vips
```
Comment (davidbyttow) [inline README.md:20]: please fix (and references below too)

**Review by davidbyttow — CHANGES_REQUESTED** [review]

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,14 @@
+module github.com/wix-playground/govips
```
Comment (davidbyttow) [inline go.mod:1]: please fix

File: resources/visual-tests.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<table>
+	<thead>
+	<th>govips</th>
+	<th>static.wixstatic.com</th>
```
Comment (davidbyttow) [inline resources/visual-tests.html:11]: not sure this file makes sense?

**Review by davidbyttow — CHANGES_REQUESTED** [review]

**Pull request closed without merging**